src/inject/index.ts

```diff
@@ -40,3 +41,8 @@ port.onDisconnect.addListener(() => {
     logWarn('disconnect');
     cleanDynamicThemeCache();
 });
+
+port.onDisconnect.addListener(watchForMediaQueryChange(() => {
```

Comment:
**we need** to listen to page event because Chrome not fires media query events for extension's `window` object

Reply:
Did you see some Chrome bug submitted?

Reply:
Yes. This code may be removed after Chrome bug would be fixed